client/src/teams/bombers.html

```diff
@@ -1,79 +0,0 @@
-<html lang="en">
```

Comment:
Remove generated file (I'm not going to comment on each of the team files individually, you get the idea).